module/VuFind/src/VuFind/ILS/Driver/KohaILSDI.php

```diff
@@ -1511,6 +1511,7 @@ public function getMyTransactions($patron)
             "GetPatronInfo&patron_id=$id" . "&show_contact=0&show_loans=1"
         );
         $end = microtime(true);
+        $requestTimes = [];
```

Comment:
This one is a rabbit hole. I'd change this to the single line `$requestTimes = [$end - $start];` to save a line of code... except I question why we are echoing out debug information inside an ILS driver at all! Shouldn't we be using $this->debug() instead... or maybe just removing this code entirely? I suspect this may be an unwanted relic that's just been in place since the driver was originally contributed many years ago. What do you think?

Reply:
I amended the commit and also change echo to debug. That is probably a relic from ancient times, and we may revisit it in the future. But I am not sure if it is worth it, as using Koha REST driver does make more sense in long term

Reply:
Thanks! I agree that this is a good enough solution for now, and because of KohaRest, it probably doesn't make sense to invest too much time in KohaILSDI, except to fix problems.